src/pyodide/internal/snapshot.ts

```diff
@@ -69,6 +69,13 @@ type LoadedSnapshotExtras = {
 
 type LoadedSnapshotMeta = SnapshotMeta & LoadedSnapshotExtras;
 
+export type PyodideEntrypointHelper = {
+  doAnImport: (mod: string) => Promise<any>;
+  cloudflareWorkersModule: any;
+  cloudflareSocketsModule: any;
+  workerEntrypoint: any;
+};
+
```

Comment:
Shouldn't this be defined in `python_entrypoint_helper.ts`?

Reply:
It'll be needed by python.ts, so I'll probably need to move it later, but sure.